### PR TITLE
[Fix] Removed warning

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6757

## Description
Removed warning and js associated with it. 

## Screencast
https://streamable.com/xi56tp

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Comment by tonytlwu** [comment]
@squallstar See if you think this is appropriate.

File: js/libs/builder.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ function generateFormDefaults(data) {
     onSubmit: [],
     template: false,
     saveProgress: true,
+    autobindProfileEditing: false,
```
Comment (squallstar) [inline js/libs/builder.js:89]: why is this required?

File: interface.html [hunk 2]
```diff
@@ -239,7 +237,7 @@ <h3>Submission</h3>
                 </div>
               </div>
             </div>
-            <div class="form-group">
+            <div  v-if="!settings.dataStore.length" class="form-group">
```
Comment (squallstar) [inline interface.html:240]: have you confirmed dataStore is always an array and you don't need an explicit check to avoid this line failing?

Comment (MaksymShokin) [reply interface.html:240]: datastore is either an empty array or an array with values

File: js/libs/builder.js (hunk 1 above)
Comment (MaksymShokin) [reply js/libs/builder.js:89]: It was actually a mistake in not having it. We use this in some places
